### PR TITLE
Converting From Log15 to ZeroLog

### PR DESCRIPTION
**Pull request by charleszqwang**

### CODE REVIEW

**Comment by charleszqwang** [comment]
test this please

File: cmd/harmony/main.go [hunk 1]
```diff
@@ -430,15 +436,16 @@ func main() {
 	if *isExplorer {
 		startMsg = "==== New Explorer Node ===="
```
Comment (LeoHChen) [inline cmd/harmony/main.go]: what's this?

File: internal/memprofiling/lib.go [hunk 2]
```diff
@@ -67,7 +69,7 @@ func (m *MemProfiling) Add(name string, v interface{}) {
 func (m *MemProfiling) Start() {
 	go m.s.ListenAndServe()
 	m.PeriodicallyScanMemSize()
-	utils.GetLogInstance().Info("Start memprofiling.")
+	utils.Logger().Info().Msg("Start memprofiling.")
```
Comment (LeoHChen) [inline internal/memprofiling/lib.go]: delete this one, no need

File: internal/utils/utils.go [hunk 3]
```diff
@@ -63,7 +63,9 @@ func ConvertFixedDataIntoByteArray(data interface{}) []byte {
 	buff := new(bytes.Buffer)
 	err := binary.Write(buff, binary.BigEndian, data)
 	if err != nil {
-		GetLogger().Crit("Failed to convert fixed data into byte array", "err", err)
+		Logger().Fatal().
```
Comment (LeoHChen) [inline internal/utils/utils.go]: don't use fatal here. use error

File: internal/utils/utils.go [hunk 4]
```diff
@@ -195,16 +197,22 @@ func LoadKeyFromFile(keyfile string) (key p2p_crypto.PrivKey, pk p2p_crypto.PubK
...
-		GetLogger().Info("No priviate key can be loaded from file", "keyfile", keyfile)
-		GetLogger().Info("Using random private key")
+		Logger().Info().
+			Str("keyfile", keyfile).
+			Msg("No private key can be loaded from file")
+		Logger().Info().Msg("Using random private key")
 		key, pk, err = GenKeyP2PRand()
 		if err != nil {
-			GetLogger().Crit("LoadKeyFromFile", "GenKeyP2PRand Error", err)
+			Logger().Fatal().
```
Comment (LeoHChen) [inline internal/utils/utils.go]: don't use fatal here. just error

**Comment by charleszqwang** [comment]
test this please

**Comment by xChickens** [comment]
Jenkins, test this please.

**Comment by charleszqwang** [comment]
test this please

**Comment by charleszqwang** [comment]
test this please

**Pull request merged**